### ISSUE TITLE
add rel_mask 

### ISSUE THREAD

**Issue by LittleTigger-alpha** [hennande/Temporal_Relational_Stock_Ranking#21]
Q1：Hello, why add rel_mask and weight？

` weight_masked = tf.nn.softmax(tf.add(rel_mask, weight), dim=0)`

and  according to equation 12 in paper, why not div rel_mask？ 
@fulifeng 
Q2：dj is the number of stocks satisfying the condition sum（a_ji) > 0.
according to pagerank， d_j is the number of stocks which has relation with stock j. and i is not euqal to the i in equation 9.  Is it right?
